src/expandProperty.js

```diff
@@ -56,12 +82,15 @@ var borderExpand = {
 }
 
 function parseFlex(value) {
-  const values = value.split(WHITESPACE_NO_CALC)
-
+  const values = splitShorthand(value)
   const longhands = {}
 
   values.forEach(val => {
-    if (val.match(LENGTH_UNIT) !== null) {
+    if (
+      val.match(LENGTH_UNIT) !== null ||
+      val.match(CALC) !== null ||
+      val.match(VAR) !== null
```

Comment:
We’re going to assume that `flexGrow` and `flexShrink` are not going to be authored as a `var(..)`, which is probably a fair assumption, yet might yield false results.